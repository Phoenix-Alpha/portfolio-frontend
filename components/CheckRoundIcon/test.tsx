import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { CheckRoundIcon } from '.'

describe('<CheckRoundIcon />', () => {
  it('should renders correct <CheckRoundIcon />', () => {
    const wrapper = shallow(<CheckRoundIcon />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
