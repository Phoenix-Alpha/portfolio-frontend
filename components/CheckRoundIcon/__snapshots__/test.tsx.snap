// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CheckRoundIcon /> should renders correct <CheckRoundIcon /> 1`] = `
<Fragment>
  <svg
    className="form-field-checkbox"
    fill="none"
    height={24}
    viewBox="0 0 24 24"
    width={24}
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      className="circle-border"
      cx="12"
      cy="12"
      r="11.5"
    />
  </svg>
</Fragment>
`;
